app/uk/gov/hmrc/gform/eval/EvaluationResults.scala

```diff
@@ -330,6 +344,32 @@ case class EvaluationResults(
       variadicValue.fold[ExpressionResult](one => nonEmpty(StringResult(one.value)))(many =>
         ExpressionResult.OptionResult(many.value)
       )
+    def atlAddresses(fcId: FormComponentId): List[String] =
+      recData.variadicFormData.distinctIndexedComponentIds(fcId.modelComponentId).map { indexedComponentId =>
+        val addressAtoms: List[ModelComponentId.Atomic] =
+          if (evaluationContext.addressLookup(indexedComponentId.baseComponentId))
+            Address.fields(indexedComponentId).filter(_.atom != Address.uk)
+          else
+            OverseasAddress.fields(indexedComponentId).toList
+
+        val variadicValues: List[Option[VariadicValue]] =
+          addressAtoms.map(atom => recData.variadicFormData.get(atom))
+        val addressLines = variadicValues.collect { case Some(VariadicValue.One(value)) if value.nonEmpty => value }
+        addressLines.mkString(", ")
+      }
+    def atlValues(fcId: FormComponentId): List[String] =
+      recData.variadicFormData
+        .forBaseComponentIdLessThen(fcId.modelComponentId)
+        .map { case (id, _) =>
+          val isHidden = exprMap.get(FormCtx(id.toFormComponentId)).fold(false)(_ === Hidden)
+          lxol.pp.log(isHidden)
```

Comment:
Please remove this log. It causes a compilation error.

Reply:
oops